.github/workflows/ci-terraform.yml

```diff
@@ -0,0 +1,21 @@
+name: Terraform Plan
+
+on:
+  push:
+    branches-ignore:
+      - master
+
```

Comment:
Need to add pull request event as well.
Since PRs can be from forks too not just branches of this repo. Can we handle that quickly?